raiden-cli/src/app.ts

```diff
@@ -0,0 +1,42 @@
+import { Server } from 'http';
+import { Logger } from 'loglevel';
+import express, { Request, Response, NextFunction, Errback } from 'express';
+import createError from 'http-errors';
+import apiV1 from './routes/api.v1';
+
+let server: Server;
+
+function notFoundHandler(_request: Request, _response: Response, next: NextFunction) {
+  next(createError(404, 'Undefined resource'));
+}
+
+function internalErrorHandler(
+  error: Errback,
+  _request: Request,
+  _response: Response,
+  next: NextFunction,
+) {
+  console.log(error); // TODO: proper error logging
```

Comment:
Please, avoid `console.*` calls directly, and prefer to use `loglevel`. Current cli version sets this as a global `Logger` object defaulting to root logger, and setting it to a account-specific logger after account is available. You could use that in an independent shared module ` let logger` export/import with some `setLogger` function to change that to the account-specific logger (which isn't much used currently but is nice to provide future options, like per-logger level and/or prepending the line with logger name, in case of multiple clients running in parallel)

Reply:
Or maybe accessing an explicit logger in some object, to avoid the global/module export.

Reply:
As I mentioned to @weilbith we already have an example for `loglevel` with `express` in the raiden-hub code.

Reply:
:heavy_check_mark: